### ISSUE TITLE
Your osu! API key is public.

### ISSUE THREAD

**Issue by Hatsuney** [firedigger/OSUAPIscripts#1]
It probably shouldn't.

**Comment by firedigger** [comment]
You are right, although it's an abandoned project, I might just as well remove the repository.
I have a lot more faith in my another osu!irc bot

**Comment by Hatsuney** [comment]
Still, someone might abuse it and peppy will stab you :(
